### ISSUE TITLE
Clicking "reorder pins" after first sign in crashes app

### ISSUE THREAD

**Issue by JibbsIsMe** [ccrama/Slide#160]
## Info
To reproduce: wipe app data, open app and sign in, and try to reorder pins. 
## Stacktrace
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{me.ccrama.redditslide/me.ccrama.redditslide.DragSort.ListViewDraggingAnim
ation}: java.lang.NullPointerException: collection == null
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
at android.app.ActivityThread.-wrap11(ActivityThread.java)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
at android.os.Handler.disp
atchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:148)
at android.app.ActivityThread.main(ActivityThread.java:5417)
at java.lang.reflect.Method.invoke(N
ative Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)Caused by: java.lang.NullPointerException: collection == null
at java.util.ArrayList.<init>(ArrayList.java:94)
at me.ccrama.redditslide.DragSort.ListViewDraggingAnim
ation.onCre
ate(ListViewDraggingAnim
ation.java:61)
at android.app.Activity.performCre
ate(Activity.java:6237)
at android.app.Instrument
ation.callActivityOnCre
ate(Instrument
ation.java:1107)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)... 9 more
```
Reported via Slide Crash Capture

**Comment by ccrama** [comment]
Oops. Will fix!

**Comment by ccrama** [comment]
Fixed with 67ea4654015083a68823c9db2addab927d3d3c9e
